lib/Backend/ValueInfo.h

```diff
@@ -228,6 +228,7 @@ class ValueInfo : protected ValueType
     bool WasNegativeZeroPreventedByBailout() const;
 
 public:
+    static bool IsEqualTo(Value * val1, Value * val2);
```

Comment:
Why is this in the ValueInfo class and not the Value class?

Reply:
No particular reason, other than other overloads were on ValueInfo. I can move it to the Value class